package.json

```diff
@@ -21,7 +21,7 @@
         "@babel/register": "^7.18.9",
         "@tsconfig/node14": "^1.0.3",
         "@types/jasmine": "3.6.2",
-        "@wdio/appium-service": "^6.12.1",
+        "@wdio/appium-service": "^7.0.0",
```

Comment:
@helenren can you downgrade it to the version it was before? I haven't push yet the version of UTAM that supports WDIO v7 officially